kymatio/scattering2d/core/scattering2d.py

```diff
@@ -1,7 +1,7 @@
 # Authors: Edouard Oyallon, Muawiz Chaudhary
 # Scientific Ancestry: Edouard Oyallon, Laurent Sifre, Joan Bruna
 
-def scattering2d(x, pad, unpad, backend, J, L, phi, psi, max_order,
+def scattering2d(x, pad, unpad, pre_pad, backend, J, L, phi, psi, max_order,
```

Comment:
Why are we adding this argument again? It's not being used in the function.

Reply:
Removed.